src/main/java/frc/robot/commands/sequences/RetractAmpSequence.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commands.sequences;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.commands.amp.DeployAmp;
+import frc.robot.commands.amp.RetractAmp;
+import frc.robot.commands.ramp.RampMove;
+import frc.robot.constants.Constants;
+import frc.robot.subsystems.Amp;
+import frc.robot.subsystems.Ramp;
+
+public class RetractAmpSequence extends ParallelCommandGroup {
+    public RetractAmpSequence(Ramp ramp, Amp amp) {
+        addCommands(
+            new RetractAmp(amp),
+            new SequentialCommandGroup(
+                new WaitCommand(0.5),
+                new RampMove(ramp, ()->-1*Constants.RAMP_ANGLE)
```

Comment:
We should create a set of target RAMP encoder positions and move to those. Going to -20 doesn't make sense, the encoder is going to read approximately from 0-26 or so.

The target for retracting the ramp here should just be 0, which you can create a constant for. Something like "RAMP_RESTING_POSITION".